src/main/java/seedu/duke/Duke.java

```diff
@@ -28,7 +28,8 @@ public static void main(String[] args) throws DukeExceptions {
                 String output = commandInput.executeCommand(projects);
                 ui.printOutput(output);
                 isExit = commandInput.isExit();
-            } catch (NullPointerException | StringIndexOutOfBoundsException | ArrayIndexOutOfBoundsException e) {
+            } catch (NullPointerException | StringIndexOutOfBoundsException | ArrayIndexOutOfBoundsException
```

Comment:
Apparently, it can't catch NullPointerException. Think you could work around it?

Reply:
ok sure but in which is the case that a null pointer exception will be thrown, prof said there is no reason we should catch it in this sectoin ryt right ?